plutus-core/src/Language/PlutusCore/Constant/Typed.hs

```diff
@@ -121,7 +128,7 @@ We need to support polymorphism for built-in functions for these reasons:
 -}
 
 {- Note [Implemetation of polymorphic built-in functions]
```

Comment:
Typo!  (I'm just pleased that I found something I could comment sensibly on...)

Reply:
That's an old one and I've accidentally fixed it in my other PR rather than this one (got lost in the branches).